src/martialarts.cpp

```diff
@@ -969,7 +969,7 @@ bool character_martial_arts::can_arm_block( const Character &owner ) const
     }
 
     // Success conditions.
-    if( owner.get_working_arm_count() >= 1 ) {
+    if( owner.blocking_score( body_part_type::type::arm ) >= 1.0f ) {
```

Comment:
```suggestion
    if( owner.blocking_score( body_part_type::type::arm ) >= 0.75f ) {
```
One arm currently contributes 0.75, not 1, so the equivalent to a 1 for working_arm_count is 0.75.

Reply:
you've got an incorrect assumption. each point of blocking score is 1 block available. each arm has 0.75 in json because i wanted them to be able to block even if they're a bit damaged.

Reply:
Previously, 1 working arm was sufficient. Now, one working arm, with the other completely nonfunctional, will give a blocking score of at most (undamaged) 0.75. Perhaps a score/max system is needed, with max normally being something like 1.5 (default to 1 block)?

Reply:
i see, this is a bit of an issue then, as a 2 of block will give you 2 possibilities to block in combat, where normall you can block once and add more blocks based on martial arts. i'm wondering if it's ok if we can keep the 2 blocks, one for each arm, and just boost the numbers to 1?

Reply:
I don't know; how much would that affect game balance re blocks?

Reply:
(talked on discord about this) basically final answer is using this as a "cap" for normal martial arts blocks. i'll have to see exactly how all of that comes together, because i seem to recall martial arts blocks can go up to several.